### PR TITLE
Add DevOps Docs

### PR DESCRIPTION
**Pull request by tellmeY18**
solves #48

### CODE REVIEW

**Comment by bodhish** [comment]
@gigincg can you review this. 
I am not sure why we need docs for the CI files. A few of it looks redundant.

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# DigitalOcean
+
+## Setting up VPC
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: Where have we documented the use of VPC requirment?

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- Navigate to the Networking tab in the DigitalOcean Console.
+- Click on the VPC section.
+- Click on the Create VPC button.
+- Choose suitable options for the VPC and click on Create VPC.
+
+## Setting up the Database
+
+- Navigate to the Databases tab in the DigitalOcean Console.
+- Click on the `Create Database` button.
+- Choose the PostgreSQL database engine.
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: Where does the user find docs for the PG version?

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- Set the VPN to the one created in the previous step.
+- Choose ideal options for the database and click on Create Database.
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: What are the "ideal options"?

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+- Click on the VPC section.
+- Click on the Create VPC button.
+- Choose suitable options for the VPC and click on Create VPC.
+
+## Setting up the Database
+
+- Navigate to the Databases tab in the DigitalOcean Console.
+- Click on the `Create Database` button.
+- Choose the PostgreSQL database engine.
+- Set the VPN to the one created in the previous step.
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: Did we create a VPN in the previous step?

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+- Choose ideal options for the database and click on Create Database.
+
+## Setting up Spaces
+
+- Navigate to the Spaces Objects Storage tab in the DigitalOcean Console.
+- Enable CDN for faster content delivery.
+- Set bucket name and set the suitable options to create the bucket.
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: What are the "sutiable options" here?

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+- Navigate to the Databases tab in the DigitalOcean Console.
+- Click on the `Create Database` button.
+- Choose the PostgreSQL database engine.
+- Set the VPN to the one created in the previous step.
+- Choose ideal options for the database and click on Create Database.
+
+## Setting up Spaces
+
+- Navigate to the Spaces Objects Storage tab in the DigitalOcean Console.
+- Enable CDN for faster content delivery.
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: I don't see an option to enable CSN after navigating to the spaces page.

File: docs/devops/Deploy/Care/digitalOcean/index.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+### 1. Deploy Care FE App
+  - Fork the [care_fe](https://github.com/ohcnetwork/care_fe) repository from [ohcnetwork](https://github.com/ohcnetwork)
+  - Select `Create App` option.
+  - Set the Service Provider as GitHub
+  - Choose `https://github.com/accountname/care_fe` as the source.
+
+  - Set the environment variables.
+    ```
+    REACT_APP_API_URL: <care-api-url>
```
Comment (bodhish) [inline docs/devops/Deploy/Care/digitalOcean/index.md]: where do I get this URl without deploying the backend?

**Review by bodhish — CHANGES_REQUESTED** [review]
I have just looked into the digital ocean docs and its incomplete. 
This is a partial review of just the DO file.

**Review by bodhish — CHANGES_REQUESTED** [review]
Lets name the files right. 
We shoudn't have files with names like `1.md`, `2.md`, `x.md` etc.

File: yarn.lock
Comment (bodhish) [inline yarn.lock:1]: Lets either use npm or yarn. Mixing both will be a pain.

File: docusaurus.config.ts [hunk 8]
```diff
@@ -70,6 +70,7 @@ const config: Config = {
         { to: "/docs/care", label: "Care", position: "left" },
         { to: "/docs/ayushma/", label: "Ayushma", position: "left" },
         { to: "/docs/leaderboard/", label: "Leaderboard", position: "left" },
+        { to: "/docs/devops/", label: "DevOps", position: "left" },
```
Comment (bodhish) [inline docusaurus.config.ts]: Move it up to line 71

File: docs/devops/index.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# DevOps Docs Overview
+
```
Comment (bodhish) [inline docs/devops/index.md:2]: Add an introduction.

**Review by bodhish — CHANGES_REQUESTED** [review]
Add an Infrastructure overview. Use mermaid diagram for building diagrams. 
Example: https://deploydocs.coronasafe.network/cloud-deployment-overview/diagram

**Comment by bodhish** [comment]
@dauntlessnomad @gigincg Please review the PR

**Comment by bodhish** [comment]
@tellmeY18 whats the status here!!!!!

**Comment by tellmeY18** [comment]
Finishing up the last bit of changes. Will commit the changes tonight.

**Comment by gigincg** [comment]
Merged as a base DevOps Doc. @tellmeY18 Do file an issue to clean this up to improve quality separately

**Pull request merged**